spec.bs

```diff
@@ -663,6 +664,11 @@ dictionary AuctionAdConfig {
 };
 </xmp>
 
+Note: To help with ease of adoption, until TBD we will support the attribute

```

Comment:
Do we have a more formal place that we can link to, that other people can follow for future updates? I'm just concerned about the case where the date comes, but we don't update the spec (because we forget maybe).

Reply:
We could also say something along the lines of "To help with ease of adoption, the browser will support this attribute while k-anonymity enforcement is being rolled out.  The attribute is not useful once k-anonymity enforcement is fully rolled out, and hence this attribute will be deprecated and removed some time after this point.  See https://developers.google.com/privacy-sandbox/relevance/protected-audience-api/k-anonymity for more up to date information."
I'd also be okay with not mentioning the deprecation stuff, and leave that for a later Intent-to-Deprecate.

Reply:
Honestly that sounds great to me, I like that wording. I think being as verbose as possible, especially regarding things that are not long for this world and are introduced solely to disappear in the future. I think that brings a lot of clarity.

Reply:
I adopted Paul's suggestion. PTAL

Reply:
Looks fine except did you check out the actual rendering? This isn't a real link: https://pr-preview.s3.amazonaws.com/brusshamilton/turtledove/pull/1177.html#:~:text=https%3A//developers.google.com/privacy%2Dsandbox/relevance/protected%2Daudience%2Dapi/k%2Danonymity.

Reply:
fixed.